src/general/base_stage/IEditorWithPatches.cs

```diff
@@ -6,4 +6,6 @@ public interface IEditorWithPatches : IEditor
     public Patch CurrentPatch { get; }
 
     public void OnCurrentPatchUpdated(Patch patch);
+
+    public void UpdateReportTabPatchSelector();
```

Comment:
This is for updating the selector when exploration state changes, right? I think it'd be better to have a more general callback for current patch being changed, which I thought would already get communicated to the editor somehow.

Reply:
This is also called by the new cheat, that's why I decided to add a new method

Reply:
I still think the name of this method should be changed to something like "OnAvailablePatchesChanged" or something like that to signal why this method is called.

Reply:
The name here is not changed yet.